source/includes/options-mongo.yaml

```diff
@@ -411,9 +411,11 @@ description: |
    .. versionadded:: 3.4
 
    .. versionchanged:: 3.6
+      \ 
 
       - Add support for :term:`zlib` compressor.
-      - Enabled by default. To disable, set to ``disabled``.
+
+      - Disabled by default in the :program:`mongo` shell.
```

Comment:
So, it was disabled in the mongo shell in 3.4.  So, this probably doesn't need to be in part of the 3.6 changes